### ISSUE TITLE
[Bug]: NanoStat version

### ISSUE THREAD

**Issue by erinyoung** [StaPH-B/docker-builds#361]
### What container were you trying to use, and how were you attempting to use it?

I'm not sure if this is something with the container or with NanoPlot itself, but there's a warning message when checking NanoPlot's version.

### Relevant log output

```shell
NanoPlot --version
/usr/local/lib/python3.6/dist-packages/nanomath/nanomath.py:132: DeprecationWarning: Call to deprecated function errs_tab.
  def ave_qual(quals, qround=False, tab=errs_tab(128)):
NanoPlot 1.33.0
```

**Comment by kapsakcj** [comment]
It's harmless. more info: https://github.com/wdecoster/NanoPlot/issues/203

Might go away if we upgrade the docker image to NanoPlot 1.40.0. Not sure if Wouter has fixed it or not

latest release on github is 1.33.0, but latest on pypi.org is 1.40.0 https://pypi.org/project/NanoPlot/

**Comment by erinyoung** [comment]
Updating it sounds like a good idea. I'll let anyone get to it before, me, though.

**Comment by kapsakcj** [comment]
working on this on the branch `cjk-nanoplot-1.40.0`. Should be ready soon

**Comment by kapsakcj** [comment]
closed via #428 . The warning is no more 👻
